plugins/backend-editor/src/Cardano/BM/Backend/Editor.lhs

```diff
@@ -113,19 +115,38 @@ instance (ToJSON a, FromJSON a) => IsBackend Editor a where
         -- local |LogBuffer|
         logbuf :: Cardano.BM.Backend.LogBuffer.LogBuffer a <- Cardano.BM.Backend.LogBuffer.realize config
 
-        thd <- Async.async $
+        thd <- Async.async $ do
             startGUI defaultConfig { jsPort       = Just port
                                    , jsAddr       = Just "127.0.0.1"
                                    , jsStatic     = Just "iohk-monitoring/static"
                                    , jsCustomHTML = Just "configuration-editor.html"
                                    } $ prepare gui config
+          `catch` nullSetup sbtrace gref
+                       EditorInternal
+                        { edSBtrace = nullTracer
+                        , edThread = thd
+                        , edBuffer = logbuf
+                        }
         Async.link thd
         putMVar gref $ EditorInternal
                         { edSBtrace = sbtrace
                         , edThread = thd
                         , edBuffer = logbuf
                         }
         return gui
+     where
+       nullSetup
+         :: Trace IO a
+         -> EditorMVar a
+         -> EditorInternal a
+         -> SomeException
+         -> IO ()
+       nullSetup trace mvar nullEditor e = do
+         meta <- mkLOMeta Error Public
+         traceWith trace $ LogObject [] meta $
```

Comment:
the context name could be `["#editor", "realize"]` so we know where this comes from.

Reply:
@CodiePP, done!